server/container_create.go

```diff
@@ -1268,6 +1395,18 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID string,
 		}
 	}
 
+	// If our user namespace doesn't own our network namespace, don't mount a fresh /sys, bind mount the host's.
+	if userNamespaceMode == pb.NamespaceMode_CONTAINER && !containerNetwork(containerConfig) {
+		specgen.RemoveMount("/sys")
+		mnt := rspec.Mount{
+			Type:        "bind",
+			Source:      "/sys",
+			Destination: "/sys",
+			Options:     []string{"ro", "rbind", "nosuid", "nodev"},
+		}
+		specgen.AddMount(mnt)
```

Comment:
Does this add a second /sys mountpoint?  If yes we should remove the first from the spec?